examples/hackernews/back-end/reactive_service/src/hackernews.service.ts

```diff
@@ -85,25 +91,22 @@ class PostsCleanupKeyMapper {
     it: NonEmptyIterator<TJSON>,
   ): Iterable<[string, TJSON]> {
     const post = it.first();
-    return [[key[1].toString(), post]];
+    return [[key[1].toString(), post] as [string, TJSON]];
```

Comment:
Why is this needed?

Reply:
No, just due to intermediate error checking, I remove it.